ubtue_Sciendo.js

```diff
@@ -44,18 +44,20 @@ function detectWeb(doc, url) {
 	}
 }
 
-function getSearchResults(doc) {
-	var items = {};
-	var found = false;
-	var rows = ZU.xpath(doc, '//li[contains(@class, "type-article")]//a[contains(@class, "text-title")]')
-	for (let i = 0; i < rows.length; i++) {
-		let href = rows[i].href;
-		let title = ZU.trimInternal(rows[i].textContent);
-		if (!href || !title) continue;
-		found = true;
-		items[href] = title;
-	}
-	return found ? items : false;
+function getSearchResults(doc, checkOnly) {
+  let items = {};
+  var found = false;
+  // TODO: adjust the CSS selector
```

Comment:
"//TODO" means the translator is not finished, so this must either be fixed or removed before a pull request can be merged